src/modules/JoinLockModule.js

```diff
@@ -0,0 +1,23 @@
+const { Module } = require('../')
+
+const Joi = require('joi')
+
+const MAX_MESSAGE_SIZE = 250
+
+// JoinLock
+module.exports = class JoinLockModule extends Module {
+  constructor (client) {
+    super('joinLock', client)
+    this.displayName = 'Join Lock'
+
+    this.defaultState = false
+    this.defaultValues = { message: '' }
+    this.specialInput = { message: { max: MAX_MESSAGE_SIZE } }
+  }
+
+  validateValues (entity) {
+    return Joi.validate(entity, Joi.object().keys({
+      message: Joi.string().max(MAX_MESSAGE_SIZE)
```

Comment:
maybe validate `.min()` and `.required()` too?

Reply:
it's not required, we have a default message

Reply:
oh yeah, and what ab `min(1)`? just for check

Reply:
if the field is empty it fallbacks to the default message, so no problem